charts/common/values.yaml

```diff
@@ -0,0 +1,600 @@
+      #suspend: true
+
+      # parallelism specifies the maximum desired number of pods the job should run at any given time. Comment out for using default (1)
+      #parallelism: 2
+      # END ONLY FOR JOBS
+
+      # START ONLY FOR CRONJOB
+      # The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron, Comment out for using default ("* * * * *")
+      #schedule: "* * * * *"
+
+      # concurrencyPolicy specifies how to treat concurrent executions of a Job. Comment out for using default (Allow)
+      #concurrencyPolicy: "Forbid"
+
+      # failedJobsHistoryLimit: The number of failed finished jobs to retain. Value must be non-negative integer. Defaults to 1.
+      #failedJobsHistoryLimit: 2
+
+      # startingDeadlineSeconds: Optional deadline in seconds for starting the job if it misses scheduled time for any reason. Missed jobs executions will be counted as failed ones.
+      #startingDeadlineSeconds: 60
+
+      # successfulJobsHistoryLimit: The number of successful finished jobs to retain. Value must be non-negative integer. Defaults to 3.
+      #successfulJobsHistoryLimit: 4
+      # END ONLY FOR CRONJOB
+
+      # start pod
+      # if a pod needs a different ServiceAccount than "default" then you can specify it here
+      # serviceAccountName: myserviceaccount
+
+      # automountServiceAccountToken indicates whether a service account token should be automatically mounted.
+      # By default, kubernetes mounts service account token at "/run/secrets/kubernetes.io/serviceaccount/".
+      # We turn this off by default as 99% of all applications simply do not communicate with the kubernetes API.
+      automountServiceAccountToken: false
+
+      # Priority indicates the importance of a Pod relative to other Pods. If a Pod cannot be scheduled,
+      # the scheduler tries to preempt (evict) lower priority Pods to make scheduling of the pending Pod possible.
+      #priorityClassName: bronze-tier-priority
+
+      # Restart policy for all containers within the pod. One of Always, OnFailure, Never. Default to Always.
+      #restartPolicy: "Never"
+
+      # podSecurityContext holds pod-level security attributes and common container settings.
+      # Some fields are also present in container.securityContext. Field values of containerSecurityContext take precedence over field values of podSecurityContext.
+      # ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#podsecuritycontext-v1-core
```

Comment:
change to general link: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

Reply:
done